src/connector/src/source/kafka/mod.rs

```diff
@@ -28,9 +28,11 @@ pub use split::*;
 
 use crate::common::KafkaCommon;
 use crate::deserialize_duration_from_string;
+pub const BROKER_REWRITE_MAP_KEY: &str = "broker.rewrite.endpoints";
```

Comment:
~~is it a breaking change?~~
& should we move it to `src/connector/src/common.rs` since it is shared by both source and sink

Reply:
moved :)